app/init.php

```diff
@@ -1,19 +1,31 @@
 <?php
+// Composer autoload
 require 'vendor/autoload.php';
+
+// Initialize Dotenv library
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+// initialize Twig template engine
 $loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
 $twigConfig = [];
-if (getenv('PRODUCTION')) {
+
+// If we're in production env, use cache
+if ($_ENV['PRODUCTION'] === true) {
     $twigConfig['cache'] = __DIR__ . '/cache';
+} else {
+    // if not production, use debug extension
+    $twigConfig['debug'] = true;
```

Comment:
To help out, I set up the debugging extension in twig- if `PRODUCTION` is `FALSE`, you will be able to use 
```
{{ dump(variable) }}
```
in templates for debugging